project-p2-layouts/code/style.css

```diff
@@ -0,0 +1,177 @@
+/* Here in the css file you'll write your styling. make sure to link the css file to the html file. If it's linked correctly you should see a light blue background. */
+
+body {
+  background-color: white;
+}
+
+.top-header {
+  font-family: "Lucida Handwritting";
```

Comment:
I don't think this font is showing, maybe you need to import it from Google fonts or something?